CONTRIBUTING.md

```diff
@@ -75,6 +82,116 @@ $ bundle exec rake compile -- --with-openssl-dir=$HOME/.openssl/openssl-3.0.1
 $ bundle exec rake test
 ```
 
+#### With OpenSSL master branch
+
+If you want to test, debug, report an issue or contribute to the Ruby OpenSSL
+or [the OpenSSL project](https://www.openssl.org/) including both non-FIPS and
+the [FIPS](https://github.com/openssl/openssl/blob/master/README-FIPS.md) cases,
+you can run the following commands to install the OpenSSL. Testing with the
+latest OpenSSL master branch is a good practice to report an issue to the
+OpenSSL project.
+
+The following configuration options are useful in this case.
+
+* `enable-fips`: Add an option to run with the OpenSSL FIPS module.
+* `enable-trace`: Add an option to enabling tracing log. You can trace logs by
+  implementing a code. See
+  [OSSL_TRACE](https://www.openssl.org/docs/man3.0/man3/OSSL_TRACE.html) for details.
+* compiler flags: You can set debugging compiler flags.
+
+```
+$ git clone https://github.com/openssl/openssl.git
+$ git branch | grep '^*'
+* master
+$ cd openssl
+$ git rev-parse --short HEAD
+0bf18140f4
+$ ./Configure \
+  --prefix=$HOME/.openssl/openssl-fips-debug-0bf18140f4 \
+  --libdir=lib \
+  shared \
```

Comment:
`shared` is enabled by default in OpenSSL >= 1.1.0 and need not be specified.

Reply:
OK. I will remove the `shared`.

Reply:
I fixed it.